packages/vuetify/src/components/VIcon/VIcon.sass

```diff
@@ -50,4 +50,4 @@
 
   &--dense
     &--is-component
-      height: $icon-size-dense
+      height: $icon-size-dense
```

Comment:
@abnersajr you have a line ending problem here. :)

Reply:
Solving.